### DISCUSSION TITLE
I need help

### DISCUSSION THREAD

**Discussion by ggfoolcool** [UseInterstellar/Interstellar#659] [Q&A] [answered]
I don't know how to get on intersteller

**Comment by replitidk** [comment]
let me help you to get a interstellar link you can go to the discord and use the link bot there and then use that link on your chrome book or you can make a code space and run in the command bar pnpm i && pnpm start and then you click the make public button and then head to the ports panel and do ctrl + click on the link in the ports panel which should then open up a new tab with the unblocker for the discord link to use the link bot is in the read me file if i am not wrong.

**Comment by replitidk** [comment]
@ggfoolcool

**Comment by replitidk** [comment]
@ggfoolcool  you got interstellar links yet?

**Reply by ggfoolcool** [reply]
no I dont

**Reply by ggfoolcool** [reply]
can you give me some

**Reply by replitidk** [reply]
no i can not but i can give you the discord link so you can get some

**Reply by replitidk** [reply]
either that or i can help you figure out how to use github codespaces or gitpod to self deploy the proxy

**Reply by ggfoolcool** [reply]
i just give me the discord link

**Reply by replitidk** [reply]
ok give me a second

**Reply by replitidk** [reply]
@ggfoolcool [discord.gg/interstellar](https://discord.gg/interstellar)

**Comment by Notplayingallday383** [comment]
someone close this bruh

**Reply by replitidk** [reply]
not yet

**Comment by Voxxie2k** [comment, marked as answer]
Close this. all the information needed is on the interstellar GitHub page. Just follow the instructions

**Reply by replitidk** [reply]
at the same time its better when someone can help someone with any questions they may have without having to bother xbubbo its why this channel in discussions is here hence the name Q&A question and answer

**Comment by replitidk** [comment]
ok yeah someone seriously close this its been like a month of it being open @ggfoolcool please close this

**Comment by nvd123YT** [comment]
bruh just open ur damn eyes
